stanford_profile_helper.module

```diff
@@ -609,11 +610,46 @@ function stanford_profile_helper_preprocess_pattern_localfooter(&$variables) {
  * Implements hook_form_FORM_ID_alter().
  */
 function stanford_profile_helper_form_media_library_add_form_embeddable_alter(array &$form, FormStateInterface $form_state) {
+
+  $source_field = $form_state->get('source_field');
+  $embed_code_field = $form_state->get('unstructured_field_name');
   $user = \Drupal::currentUser();
   $authorized = $user->hasPermission('create field_media_embeddable_code')
     || $user->hasPermission('edit field_media_embeddable_code');
 
-  if (isset($form['container']['field_media_embeddable_code'])) {
-    $form['container']['field_media_embeddable_code']['#access'] = $authorized;
+  if (isset($form['container'][$embed_code_field])) {
+    $form['container'][$embed_code_field]['#access'] = $authorized;
+  }
+
+  if (isset($form['container'][$source_field])) {
```

Comment:
should also check if the user does not have access to the embed code field.

Reply:
That happens above on line 620.  I changed it to discover the field names, so it uses $embed_code_field rather than the field name directly.  All users have access to the $source_field, which is the oEmbed field.

Reply:
this changes help documentation all the time regardless if the user sees the `$embed_code_field` or not. We only really need to inform a user if they cant put in the raw embed code.

Reply:
Ok.  I set up a separate check, because either way, the title has to change on this form. I put the same check into the other form as well.